src/main/include/subsystems/Turret.hpp

```diff
@@ -150,7 +150,7 @@ class Turret : public ControlledSubsystemBase<2, 1, 1> {
 
     void AutonomousInit() override {
         Enable();
-        SetControlMode(TurretController::ControlMode::kAutoAim);
+        // SetControlMode(TurretController::ControlMode::kAutoAim);
```

Comment:
Should the turret start in vision-aim mode instead? It should still be set explicitly to avoid mistakes.